tests/sles4sap/trento/test_trento_deploy.pm

```diff
@@ -10,7 +10,7 @@ use Mojo::Base 'publiccloud::basetest';
 use base 'consoletest';
 use testapi;
 use serial_terminal 'select_serial_terminal';
-use utils qw(script_retry);
+use utils 'script_retry';
```

Comment:
This is not necessary: https://github.com/os-autoinst/os-autoinst/pull/1375

But you can leave it if you want.